src/test_suite/test_suite.ts

```diff
@@ -307,11 +308,13 @@ export class TestCase {
                 const observed = formatCart(state.cart, catalog);
                 orders.push(observed);
 
-                // Compare observed Orders
-                const expected = this.expected[i];
-                succeeded = isomorphic
-                    ? ordersAreEqualCanonical(observed, expected)
-                    : ordersAreEqual(observed, expected);
+                if (succeeded && (evaluateIntermediate || this.inputs.length - 1 === i)) {
```

Comment:
Generally, I like putting the variable being compared before that which it is being compared to. So, here I would have `i === this.inputs.length -1`.

[This pattern is referred to as Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions), which is a hilarious name~

Reply:
make this change, i will